libs/template/presets/storefront/experience/footer.ts

```diff
@@ -86,7 +86,7 @@ const aboutUs = {
   content: {
     data: {
       text: `<b>About us</b><br/>
-        <a href="/">Company</a><br/>
+        <a href="/about/company">Company</a><br/>
```

Comment:
bc

Reply:
I'd leave it out for now. Whenever our CMS integration is ready, we'll create the footer navigations by CMS, e.g. getting a content list where we query CMS. We'll have different lists from different CMS.